postgres-appliance/Dockerfile

```diff
@@ -214,7 +216,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                 EXTRA_EXTENSIONS=""; \
             fi \
 \
-            && for n in bg_mon-${BG_MON_COMMIT} pg_auth_mon-${PG_AUTH_MON_COMMIT} set_user $EXTRA_EXTENSIONS; do \
+            && for n in bg_mon-${BG_MON_COMMIT} pg_auth_mon-${PG_AUTH_MON_COMMIT} pg_mon-${PG_MON_COMMIT} set_user $EXTRA_EXTENSIONS; do \
```

Comment:
Lets move it [here](https://github.com/zalando/spilo/pull/420/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R211)